inc/head.php

```diff
@@ -29,6 +30,11 @@
 
     $kmwbuild = $kmw_builds[$tier];
   }
+
+  if(version_compare($kmw_builds['stable'], $kmw_builds['beta']) < 0) {
+    $hasBeta = true;
+  }
+
```

Comment:
Compare against https://github.com/keymanapp/keymanweb.com/blob/f3b1187d9821adc2da7ebd038f90d8fbf3f86a18/inc/head.php#L24-L28

Why are these separate?  Just define `$hasBeta` in the block above.

Reply:
That, or move the check above the main `if` and use `$hasBeta` to set the default tier appropriately, instead of the (unchanged) block mentioned above.